extensions/amp-consent/0.1/test-e2e/test-amp-consent-server-side-expire-cache.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-consent',
+  {
+    testUrl:
+      'http://localhost:8000/test/manual/amp-consent/amp-consent-basic-uses.amp.html#amp-geo=mx',
+    experiments: ['amp-consent-geo-override'],
+    // TODO (micajuineho): Add shadow-demo after #25985 is fixed, and viewer-demo when...
+    environments: ['single'],
+  },
+  env => {
+    let controller;
+    let ui1;
+    let ui2;
+    let postPromptUi;
+    let tillResponded;
+    let accepted;
+    let autoReject;
+    let defaultBlock;
+    let notBlocked;
+    let twitter;
+    let requestBank;
+
+    beforeEach(() => {
+      controller = env.controller;
+      requestBank = env.requestBank;
+    });
+
+    async function findElements() {
```

Comment:
just curious what's a good way to share helper method between multiple test files. cc @estherkim

Reply:
You can create something like `common.js` in the same dir for the `amp-consent` tests to import?

Reply:
Awesome! @micajuine-ho can we create a helper method to make e2e tests smaller : )

Reply:
Ok! Sounds good. Thanks @estherkim